src/main/webapp/css/user-page.css

```diff
@@ -40,8 +40,14 @@
   margin: 5px;
 }
 
+/* Restrict image size */
+.message-body img {
+  display: block;
+  margin-top: 25px;
+  max-width: 75%;
```

Comment:
Will height also be responsive with these restrictions? It can probably be something to add for future PR's